app/javascript/dashboard/mixins/agentMixin.js

```diff
@@ -7,34 +7,52 @@ export default {
         this.inboxId
       );
     },
-    ...mapGetters({ currentUser: 'getCurrentUser' }),
+    ...mapGetters({
+      currentUser: 'getCurrentUser',
+    }),
     isAgentSelected() {
       return this.currentChat?.meta?.assignee;
     },
+    createNoneAgent() {
+      return {
+        confirmed: true,
+        name: 'None',
+        id: 0,
+        role: 'agent',
+        account_id: 0,
+        email: 'None',
+      };
+    },
     agentsList() {
       const agents = this.assignableAgents || [];
-      return [
-        ...(this.isAgentSelected
-          ? [
-              {
-                confirmed: true,
-                name: 'None',
-                id: 0,
-                role: 'agent',
-                account_id: 0,
-                email: 'None',
-              },
-            ]
-          : []),
-        ...agents,
-      ].map(item =>
-        item.id === this.currentUser.id
-          ? {
-              ...item,
-              availability_status: this.currentUser.availability_status,
-            }
-          : item
-      );
```

Comment:
@muhsin-k I had to remove that specific code section since the `availability_status` was becoming undefined.
<img width="398" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/4546adca-ccb9-4982-98ea-da654c325512">

Reply:
Please check whether the API returns the field `availability_status`.

Reply:
@muhsin-k This is what the API is returning
<img width="249" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/05c62ab8-12df-4f6e-aca2-ebbf941edc55">

Reply:
@iamsivin The `availability_status` was moved to accounts.

![CleanShot 2023-05-23 at 21 39 23](https://github.com/chatwoot/chatwoot/assets/12408980/ffdaa61d-85ee-4d78-88ef-1e91c8fc16ee)

Reply:
Done https://github.com/chatwoot/chatwoot/pull/7174/commits/55945c57138739f82d35064544bf9b29b5d60dae